package.json

```diff
@@ -8,6 +8,7 @@
     "url": "https://github.com/elastic/apm-agent-js-base.git"
   },
   "scripts": {
+    "release": "lerna publish",
```

Comment:
How can we still tell it to release only one package?

Imagine there're changes in both packages but we only want to release one of the at the moment.

Reply:
The changes are automatically picked up, So if changes are in js-core both would be published.. if the changes are only on js-base, Only the js-base would be published... 

I will work on individual releasing and docs(Release.md) in separate PR.

Reply:
Thanks! looking forward to that one then 👍